RecoHGCal/TICL/plugins/PatternRecognitionbyFastJet.cc

```diff
@@ -278,7 +269,7 @@ void PatternRecognitionbyFastJet<TILES>::energyRegressionAndID(const std::vector
   }
 
   // run the inference (7)
-  tensorflow::run(eidSession_, inputList, outputNames, &outputs);
+  tensorflow::run(const_cast<tensorflow::Session *>(eidSession), inputList, outputNames, &outputs);
```

Comment:
`eidSession` is already a `const tensorflow::Session *`: why do you need the `const_cast`?